src/boot/efi/devicetree.c

```diff
@@ -1,5 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <endian.h>
```

Comment:
This header is not provided by the compiler and therefore not guaranteed to be safe to use in freedstanding env.

Reply:
On second thought, what should the approach here be? Should be32toh() be defined just for this purpose (just on this file) or should it go somewhere in src/fundamental?

Reply:
Since it's the only user, you can keep it in the file and just define it with `__builtin_bswap*` directly. UEFI is little endian exclusively, so no point in dealing with the other case.